templates/zerver/help/import-from-slack.md

```diff
@@ -0,0 +1,50 @@
+# Import from Slack
+
+To import users and channels from Slack to Zulip please follow
```

Comment:
You can use the `{!follow-steps.md!}` macro here, like this: 
```
{!follow-steps.md!} import users and channels from Slack to Zulip.
```

You might also want to provide a link to Slack.